department-of-veterans-affairs/caseflow
# Queue | Update loadedQueue issues from server after CUD

Resolves #5200 

### Description
When the user makes changes to issues during the appeal checkout flow, `AddEditIssueView` saves those changes to the server immediately. This updates `loadedQueue`, the front-end source of truth, with issue changes from the server.

### Acceptance Criteria 
- [x] Changes to appeal issue count is reflected in `AttorneyTaskTable` view
- [x] Changes to appeal issue count/attributes are reflected in `AppealDetail` view

### Testing Plan
1. Go to Draft Decision checkout flow for appeal
2. Create/delete/update an issue while selecting dispositions. Cancel checkout flow (navigate back to `/queue` or `/queue/tasks/${vacols_id}`) by clicking link on page (to prevent reloading), confirm issue changes are displayed

Before:
![issue_count_doesnt_update](https://user-images.githubusercontent.com/8533004/39020557-62e4b5e8-43fb-11e8-803f-202a78a477e2.gif)


After
![create_issue_count](https://user-images.githubusercontent.com/8533004/39020518-2d36f942-43fb-11e8-9060-633e440addb9.gif)
